index.html

```diff
@@ -39,7 +39,7 @@ <h3 class="text-muted">Chromebrew</h3>
       </div>
 
       <div class="row marketing">
-        <div class="col-lg-6">
+        <div class="col-6">
```

Comment:
Thanks for modifying that.  I also need some sense that I fee I understand and agree with in order to merge it by my name.  This is small change but important for me to agree with this PR.  Thanks a lot.